internal/test/test.go

```diff
@@ -10,6 +10,7 @@ import (
 // CmdValidator helps validate a cobra.Command, verifying the number of sub commands
 // and the flags that are being defined for it
 func CmdValidator(t *testing.T, subject *cobra.Command, nSubCommands int, flags []string) {
+	t.Helper()
```

Comment:
[q] what the purpose of this piece of code?

Reply:
https://golang.org/pkg/testing/#T.Helper